frontend/dispatcher.lua

```diff
@@ -332,11 +321,42 @@ function Dispatcher:init()
     for i=1,#CreOptions do
         parseoptions(CreOptions, i)
     end
+    UIManager:broadcastEvent(Event:new("DispatcherRegisterActions"))
     Dispatcher.initialized = true
 end
 
+--[[--
+Adds settings at runtime.
+
+@usage
+    function Hello:onDispatcherRegisterActions()
+        Dispatcher:registerAction("helloworld_action", {category="none", event="HelloWorld", title=_("Hello World"), filemanager=true,})
+    end
+
+    function Hello:init()
+        self:onDispatcherRegisterActions()
+    end
+
+
+@param name
+the key to use in the table
```

Comment:
Not strictly necessary, but I find this multiline thing a bit hard to read. Same for the rest.
```suggestion
@param name the key to use in the table
```